File: app/src/main/java/com/banglalink/toffee/listeners/OnPlayerControllerChangedListener.java
package com.banglalink.toffee.listeners;

/**
 * Created by shantanu on 5/5/17.
 */

public interface OnPlayerControllerChangedListener {
    boolean onPlayButtonPressed(int currentState);
    boolean onFullScreenButtonPressed();
    boolean onDrawerButtonPressed();
    boolean onMinimizeButtonPressed();
    boolean onOptionMenuPressed();
    boolean onShareButtonPressed();
    void  onPlayerIdleDueToError();
    void onRotationLock(boolean isAutoRotationEnabled);
    boolean onSeekPosition(int position);
    void onControllerVisible();
    void onControllerInVisible();
}

File: app/src/main/java/com/banglalink/toffee/listeners/OnPlayerControllerChangedListener.kt
package com.banglalink.toffee.listeners

interface OnPlayerControllerChangedListener {
    fun onPlayButtonPressed(currentState: Int): Boolean
    fun onFullScreenButtonPressed(): Boolean
    fun onDrawerButtonPressed(): Boolean
    fun onMinimizeButtonPressed(): Boolean
    fun onOptionMenuPressed(): Boolean
    fun onShareButtonPressed(): Boolean
    fun onPlayerIdleDueToError()
    fun onRotationLock(isAutoRotationEnabled: Boolean)
    fun onSeekPosition(position: Int): Boolean
    fun onControllerVisible()
    fun onControllerInVisible()
    fun onMediaItemChanged(){}
}